modulefiles/ufs_common.lua

```diff
@@ -6,21 +6,21 @@ local ufs_modules = {
   {["jasper"]          = "2.0.32"},
   {["zlib"]            = "1.2.13"},
   {["libpng"]          = "1.6.37"},
-  {["hdf5"]            = "1.14.0"},
+  {["hdf5"]            = "1.14.3"},
   {["netcdf-c"]        = "4.9.2"},
   {["netcdf-fortran"]  = "4.6.1"},
-  {["parallelio"]      = "2.5.10"},
-  {["esmf"]            = "8.6.0"},
-  {["fms"]             = "2024.01"},
+  {["parallelio"]      = "2.6.2"},
+  {["esmf"]            = "8.8.0"},
+  {["fms"]             = "2024.02"},
   {["bacio"]           = "2.4.1"},
-  {["crtm"]            = "2.4.0"},
+  {["crtm"]            = "2.4.0.1"},
   {["g2"]              = "3.5.1"},
   {["g2tmpl"]          = "1.13.0"},
-  {["ip"]              = "4.3.0"},
+  {["ip"]              = "5.1.0"},
   {["sp"]              = "2.5.0"},
   {["w3emc"]           = "2.10.0"},
-  {["gftl-shared"]     = "1.6.1"},
-  {["mapl"]            = "2.40.3-esmf-8.6.0"},
+  {["gftl-shared"]     = "1.9.0"},
+  {["mapl"]            = "2.53.4-esmf-8.8.0"},
   {["scotch"]          = "7.0.4"},
```

Comment:
On Ursa are we planning to use 7.0.7 or 7.0.4 for SCOTCH? 

If i want to run on ursa and look at the wave failures, is this branch up to date? Should I be switching this to 7.0.7 or are there other updates needed and I should wait?

Reply:
Thanks @ulmononian  -  What code changes exactly are needed on my branch for that to run?  I'll test things out on my end, but I wasn't expecting my WW3 branch to require any changes for this.